microbench/src/main/java/io/netty/microbench/channel/epoll/EpollSocketChannelBenchmark.java

```diff
@@ -136,4 +136,12 @@ public void tearDown() throws Exception {
     public Object pingPong() throws Exception {
         return chan.pipeline().writeAndFlush(abyte.retainedSlice()).sync();
     }
+
+    @Benchmark
+    public Object execute() throws Exception {
```

Comment:
Reuse the runnable to reduce the noise of GC if possible

Reply:
Good point, done!